privval/signer_listener_endpoint.go

```diff
@@ -207,6 +223,17 @@ func (sl *SignerListenerEndpoint) serviceLoop() {
 	}
 }
 
+func (sl *SignerListenerEndpoint) isAllowedAddr(addr net.Addr) bool {
+	if len(sl.allowAddr) == 0 {
+		return true
+	}
+	if strings.Contains(addr.String(), ":") {
+		addrOnly := addr.String()[:strings.Index(addr.String(), ":")]
```

Comment:
It appears to use a colon `:` as an address delimiter and use the first one of the addresses, doesn't it?
In IPv6 environments, this delimiter doesn't seem to work correctly since the format is `2001:0db8:bd05:01d2:288a:1fc0:0001:10ee`.
Therefore, I think it would be better to use comma `,` or semi-colon `;` if you intend to use some delimiter.

Reply:
addr.String() returns like this "192.168.0.5:123456".

I didn't consider IPv6. Let me try to check what addr.String() returns for IPv6`2001:0db8:bd05:01d2:288a:1fc0:0001:10ee`.
I may need to find other ways to get address part only. let me check

Reply:
Ah, I misunderstood the input and configuration in reverse😅 The colon is meant to be a separator between IP address and port number, right? However, in such that IPv6 environment, the string would be something like `[2001:db8::1]:8080`, which would cause the same problem.

Reply:
Yes, you're right. I need to consider IPv6. Thanks, I'll fix it.